packages/appstore/src/components/onboarding/onboarding.scss

```
@@ -0,0 +1,43 @@
+.onboarding-wrapper {
+    height: 100%;
+    background-color: $color-grey-7;
+}
+
+.onboarding-header {
+    display: flex;
+    justify-content: space-between;
+
+    &__cross-icon {
+        margin-top: 3.5rem;
+        margin-right: 3rem;
+    }
+}
+
+.onboarding-body {
+    display: flex;
+    justify-content: center;
+    color: $color-white;
+}
+
+.onboarding-footer {
+    background-color: $color-white;
```

You'll need to update all the colors to variables in `Themes.scss` to handle dark theme as well.
```suggestion
    background-color: var(--general-main-1);
```